Cargo.lock

```diff
@@ -4,1743 +4,1729 @@
 name = "adler"
```

Comment:
I wouldn't expect this file to change unless the version of Rust is changing too (note that this may introduce other issues in compiling the Docker container.)

Can you either undo these changes, or upgrade the version of Rust in [this file](https://github.com/sourcegraph/syntect_server/blob/master/rust-toolchain) and the Dockerfile?